mindjunkies/courses/templatetags/courses_tags.py

```
@@ -19,3 +19,7 @@
 @register.filter
 def subtract(value, arg):
     return int(value) - int(arg)
+
+@register.filter(name="add_class")
+def add_class(field, css_class):
+    return field.as_widget(attrs={"class": css_class})
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)